manifests/init.pp

```diff
@@ -541,6 +541,16 @@
 #
 #
 class kubernetes (
+  Enum['archive','package'] $kubernetes_install_method  = 'package',
+  Optional[String] $kubernetes_archive                  = "kube-bins-${kubernetes_version}.tar.gz",
```

Comment:
If an empty string doesn't make sense, please switch to `String[1]`

Reply:
done